linter.py

```diff
@@ -1,15 +1,60 @@
+import sublime, sublime_plugin, subprocess, os, os.path, sys
 from SublimeLinter.lint import NodeLinter
 
+BASE_PATH = os.path.abspath(os.path.dirname(__file__))
+CODE_DIRS = [
+  'plugin_helpers'
+]
+sys.path += [BASE_PATH] + [os.path.join(BASE_PATH, f) for f in CODE_DIRS]
+
+from plugin_helpers.utils import memoize
+
+class SettingManager(object):
+    @memoize
```

Comment:
Why are you memoizing the settings? That means if the user updates settings they'll have to restart Sublime, right? That's definitely not what they would expect.